ansible/roles/ipsilon/tasks/main.yml

```diff
@@ -110,6 +111,31 @@
     - httpd_can_network_connect
     - httpd_use_fusefs
 
+- name: Remove default static alias from httpd conf
+  replace:
+    path: /etc/ipsilon/idp/idp.conf
+    regexp: 'Alias \/idp\/ui \/usr\/share\/ipsilon\/ui'
+    replace: '#Alias /idp/ui /usr/share/ipsilon/ui'
```

Comment:
This is not idempotent, i.e. when reprovisioning it would comment the commented line again and so forth. Prepend `^\s*`  to the regex?